ign_ros2_control_demos/urdf/test_cart_position.xacro.urdf

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" ?>
```

Comment:
why only this file is here ?

Reply:
all launch files except for ign_ros2_control_demos/launch/cart_example_position.launch.py are pointing to the launch files in gz_ros2_control_demos. I let this one here (used by the cart-position launch file) to test with gz_ros2_control_tests/tests/position_test_backward_compatibility.py if the new gz_ros2_control_plugin also works with
```xml
    <hardware>
      <plugin>ign_ros2_control/IgnitionSystem</plugin>
    </hardware>
```
```xml
        <plugin filename="ign_ros2_control-system" name="ign_ros2_control::IgnitionROS2ControlPlugin">
      <parameters>$(find gz_ros2_control_demos)/config/cart_controller_position.yaml</parameters>
    </plugin>
```